### PR TITLE
GPG-155 Fix few tables accessibility issues

### PR DESCRIPTION
**Pull request by aidayounesi**

### CODE REVIEW

File: GenderPayGap.WebUI/Views/AdminOrganisationAddress/ConfirmAddressChange.cshtml [hunk 1]
```diff
@@ -64,7 +64,7 @@
             <table class="govuk-table">
                 <thead class="govuk-table__head">
                     <tr class="govuk-table__row">
-                        <th scope="col" class="govuk-table__header"></th>
+                        <th scope="col" class="govuk-table__header govuk-visually-hidden">Address component</th>
```
[the remark this answers is not in the document]
Comment (aidayounesi) [reply GenderPayGap.WebUI/Views/AdminOrganisationAddress/ConfirmAddressChange.cshtml:67]: Sure, thanks for noticing

File: GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml [hunk 2]
```diff
@@ -92,45 +89,59 @@
         else
         {
             <table class="govuk-table">
+                <thead class="govuk-table__head govuk-visually-hidden">
+                    <tr class="govuk-table__row">
+                        <th scope="col" class="govuk-table__header govuk-!-padding-0">
+                            <span class=" govuk-visually-hidden">SIC codes to add</span>
```
Comment (jamesgriff) [inline GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml:95]: You might not need this span if you're using `govuk-visually-hidden` on the `thead` (just put the text directly inside the `th`)?

File: GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml [hunk 3]
```diff
@@ -92,45 +89,59 @@
...
-        <h3 class="govuk-heading-m">SIC codes to keep</h3>
-
+        <h3 class="govuk-heading-m govuk-!-margin-bottom-2">
+            SIC codes to keep
+        </h3>
         @if (Model.SicCodesToKeep.Count == 0)
         {
             <p class="govuk-body">
-                No SIC codes to keep
+                No SIC codes to add
```
Comment (jamesgriff) [inline GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml:124]: I think this should still say `No SIC codes to keep`

File: GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml [hunk 4]
```diff
@@ -92,45 +89,59 @@
...
         else
         {
             <table class="govuk-table">
+                <thead class="govuk-table__head">
+                    <tr class="govuk-table__row">
+                        <th scope="col" class="govuk-table__header govuk-!-padding-0">
+                            <span class=" govuk-visually-hidden">SIC codes to keep</span>
+                        </th>
+                    </tr>
+                </thead>
```
Comment (jamesgriff) [inline GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml:136]: Would it be worth moving the `govuk-visually-hidden` from the `span` to the `thead` (and maybe removing the `span` altogether (you could put the text directly inside the `th`))?
If you think it's better here, then happy to discuss

File: GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml [hunk 5]
```diff
@@ -153,26 +164,32 @@
         else
         {
             <table class="govuk-table">
+                <thead class="govuk-table__head">
+                    <tr class="govuk-table__row">
+                        <th scope="col" class="govuk-table__header govuk-!-padding-0">
+                            <span class=" govuk-visually-hidden">SIC codes to remove</span>
+                        </th>
+                    </tr>
+                </thead>
```
Comment (jamesgriff) [inline GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml:173]: Same - happy to discuss

File: GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/OfferNewCompaniesHouseSicCodes.cshtml [hunk 6]
```diff
@@ -78,23 +78,32 @@
...
+            <h3 class="govuk-heading-m govuk-!-margin-bottom-2">
+                Current SIC codes
+            </h3>
             @if (!currentSicCodes.Any())
             {
-                <div class="govuk-body">
+                <p class="govuk-body">
                     No current SIC codes
-                </div>
+                </p>
```
Comment (jamesgriff) [inline GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/OfferNewCompaniesHouseSicCodes.cshtml:95]: This is nicer as a `p` - good spot 😄

File: GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/OfferNewCompaniesHouseSicCodes.cshtml [hunk 7]
```diff
@@ -78,23 +78,32 @@
...
             else
             {
                 <table class="govuk-table">
+                    <thead class="govuk-table__head">
+                        <tr class="govuk-table__row">
+                            <th scope="col" class="govuk-table__header govuk-!-padding-0">
+                                <span class=" govuk-visually-hidden">Current SIC codes</span>
+                            </th>
+                        </tr>
+                    </thead>
```
Comment (jamesgriff) [inline GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/OfferNewCompaniesHouseSicCodes.cshtml:106]: Same - happy to discuss

File: GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/OfferNewCompaniesHouseSicCodes.cshtml [hunk 8]
```diff
@@ -113,16 +122,25 @@
...
             else
             {
                 <table class="govuk-table">
+                    <thead class="govuk-table__head">
+                        <tr class="govuk-table__row">
+                            <th scope="col" class="govuk-table__header govuk-!-padding-0">
+                                <span class=" govuk-visually-hidden">SIC codes from Companies House</span>
+                            </th>
+                        </tr>
+                    </thead>
```
Comment (jamesgriff) [inline GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/OfferNewCompaniesHouseSicCodes.cshtml:143]: Same - happy to discuss

File: GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml (hunk 2 above)
Comment (aidayounesi) [reply GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml:95]: I think it looks nicer with the line above the first row and if I add class visually hidden to <th> then that line would be invisible.

File: GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml (hunk 3 above)
Comment (aidayounesi) [reply GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml:124]: Good catch thank you

File: GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml (hunk 2 above)
Comment (jamesgriff) [reply GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml:95]: Ah ok, that's fair enough 😄

**Pull request merged**